Sources/SWBTestSupport/RunDestinationTestSupport.swift

```diff
@@ -328,7 +328,10 @@ extension RunDestinationInfo {
         case .elf:
             environment.prependPath(key: "LD_LIBRARY_PATH", value: toolchain.path.join("usr/lib/swift/\(platform)").str)
         case .pe:
-            environment.prependPath(key: .path, value: core.developerPath.path.join("Runtimes").join(toolchain.version.description).join("usr/bin").str)
+            let currentPath = Environment.current[.path] ?? ""
```

Comment:
Why did this change to start reading from the current environment instead of the given one? This doesn't seem right. We should be aiming to reduce usage of globals throughout the codebase as much as possible.

Also, why was `throws` added? Nothing in here throws.